src/modules/dashboard/components/Members/MembersTitle.tsx

```diff
@@ -78,66 +88,78 @@ const MembersTitle = ({
     (e.target as HTMLInputElement).placeholder = '';
   }, []);
 
+  const isMobile = useMediaQuery({ query });
   return (
     <div className={styles.titleContainer}>
       <div className={styles.titleLeft}>
         <Heading
           text={MSG.title}
           appearance={{ size: 'medium', theme: 'dark' }}
+          textValues={{ isMobile }}
         />
```

Comment:
All of the below is just adding the `ColonyDomainSelector` when  `isMobile` is true